spec.md

```diff
@@ -16,6 +16,8 @@ The key words "unspecified", "undefined", and "implementation-defined" are to be
 An implementation is not compliant if it fails to satisfy one or more of the MUST, REQUIRED, or SHALL requirements for the protocols it implements.
 An implementation is compliant if it satisfies all the MUST, REQUIRED, and SHALL requirements for the protocols it implements.
 
+Scalar fields, even REQUIRED ones, will be defaulted if not specified, and any field set to the defaul value will not be serialized over the wire as per [proto3](https://developers.google.com/protocol-buffers/docs/proto3#default).
```

Comment:
Wording looks good. But can we move this to a section right after the [Size Limits section](https://github.com/container-storage-interface/spec/blob/master/spec.md#size-limits)?

Reply:
Sure, it does makes more sense to be in the Required vs Optional section.